outscale/resource_outscale_net.go

```diff
@@ -71,7 +71,19 @@ func resourceOutscaleOAPINetCreate(d *schema.ResourceData, meta interface{}) err
 
 	d.SetId(resp.Net.GetNetId())
 
-	return resourceOutscaleOAPINetRead(d, meta)
+	return resource.Retry(120*time.Second, func() *resource.RetryError {
+		err = resourceOutscaleOAPINetRead(d, meta)
+		if err != nil {
+			return resource.NonRetryableError(err)
+		}
+		if c, ok := d.GetOk("state"); ok {
+			state := c.(string)
+			if 0 != strings.Compare(state, "available") {
```

Comment:
I don't think we use yoda condition in the provider, so could you use
```
if strings.Compare(state, "available") != 0 {
```

Reply:
Correction done